src/cve/pipeline/pipeline.py

```diff
@@ -37,7 +37,7 @@
 from morpheus_llm.stages.llm.llm_engine_stage import LLMEngineStage
 
 from ..data_models.config import RunConfig
-from ..data_models.input import AgentMorpheusEngineInput
+from ..data_models.input import AgentMorpheusEngineInput, AgentMorpheusInput
```

Comment:
The additional import is no longer needed
```suggestion
from ..data_models.input import AgentMorpheusEngineInput
```